fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTabLayouts.kt

```diff
@@ -37,6 +37,9 @@ import org.mozilla.fenix.theme.FirefoxTheme
  *
  * @param tabs The list of [TabSessionState] to display.
  * @param displayTabsInGrid Whether the tabs should be displayed in a grid.
+ * @param selectedTabId The ID of the currently selected tab.
+ * @param selectionMode [TabsTrayState.Mode] indicating whether the Tabs Tray is in single selection
+ * or multi-selection and contains the set of selected tabs.
```

Comment:
I don't think this is the place the solve this problem, but I feel like "selected" is annoyingly overloaded between the current active tab and the (multi-) selected tabs. "Selected" actually makes more sense here IMO, so I think this would mean updating the BrowserState to use a different name for the (last) active tab.

Reply:
I think we're in agreement here. This is also evident with `TabListItem` and `TabGridItem` currently having:
```
isSelected: Boolean = false,
multiSelectionEnabled: Boolean = false,
multiSelectionSelected: Boolean = false,
```

I can file a ticket for a post-release or "or when time allows" follow-up.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1828527